Sample/Cloud/Configs/devicecontent.json

```diff
@@ -12,23 +12,23 @@
           {
             "required": "true",
             "valueExpression": "$.heartRate",
-            "valueName": "hr"
+            "valueName": "heartRate"
           }
         ]
       }
     },
     {
       "templateType": "JsonPathContent",
       "template": {
-        "typeName": "steps",
+        "typeName": "stepCount",
         "typeMatchExpression": "$..[?(@stepCount)]",
         "deviceIdExpression": "$.deviceId",
         "timestampExpression": "$.endDate",
         "values": [
           {
             "required": "true",
             "valueExpression": "$.stepCount",
-            "valueName": "steps"

```

Comment:
Just curious the reason for the change?  It potentially could break anyone if they take one updated template but not the other.

Reply:
That's a good point. Was hoping to make the template syntax consistent between all templates, but I think I will just revert this change.